usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -0,0 +1,141 @@
+DROP VIEW IF EXISTS award_delta_view;
+
+CREATE VIEW award_delta_view AS
+WITH views AS (
+    SELECT mv_contract_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_contract_award_search INNER JOIN awards a ON
+            a.id = mv_contract_award_search.award_id
+    UNION
+    SELECT mv_grant_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_grant_award_search INNER JOIN awards a ON
+            a.id = mv_grant_award_search.award_id
+    UNION
+    SELECT mv_directpayment_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_directpayment_award_search INNER JOIN awards a ON
+         a.id = mv_directpayment_award_search.award_id
+    UNION
+    SELECT mv_idv_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_idv_award_search INNER JOIN awards a ON
+            a.id = mv_idv_award_search.award_id
+    UNION
+    SELECT mv_loan_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_loan_award_search INNER JOIN awards a ON
+            a.id = mv_loan_award_search.award_id
+    UNION
+    SELECT mv_other_award_search.*, a.generated_unique_award_id, a.update_date
+        FROM mv_other_award_search INNER JOIN awards a ON a.id = mv_other_award_search.award_id
+)
+SELECT
+  DISTINCT views.award_id,
+  views.generated_unique_award_id,
+    CASE
+    WHEN views.type IN ('02', '03', '04', '05', '06', '10', '07', '08', '09', '11') AND views.fain IS NOT NULL THEN views.fain
+    WHEN views.piid IS NOT NULL THEN views.piid  -- contracts. Did it this way to easily handle IDV contracts
+    ELSE views.uri
+  END AS display_award_id,
```

Comment:
I'm not sure we want to propagate this logic further. It is trivial to compute which value to return in the view business logic, like how it is currently done in `spending_by_award.py` LN 159 - 161

Reply:
AH.... the sorting. dang it

Reply:
Can ES do multi-field sorting?

Reply:
yeah it can

Reply:
Unless I am mistaken, the ES query could sort the results by the three fields (PIID, FAIN, URI) when requested by the API and the view business logic could return the desired value which would eliminate the need for `display_award_id`